tests/templates/script/commonPartials.js

```diff
@@ -0,0 +1,62 @@
+const path = require('path');
+const fs = require('fs');
+const hbs = require('../../test-utils/hbs');
+const directoryPath = 'templates/common-partials/script'
+const partialPath = path.resolve(__dirname, `../../../${directoryPath}`);
+const files = fs.readdirSync(partialPath);
+
+/*
+//Generates common-partials default values
```

Comment:
this is pretty much the same thing as below but replacing the expect with a writeFile, so we don't need to keep this around.